src/opera/instance/topology.py

```diff
@@ -53,5 +53,12 @@ def undeploy(self, verbose, workdir, num_workers=None):
     def write(self, data, instance_id):
         self.storage.write_json(data, "instances", instance_id)
 
+    def write_all(self):
+        for node in self.nodes.values():
+            node.write()
```

Comment:
How does that `write_all` change the current behavior of opera storage?

Reply:
It does not. It is just a simplification, in case we want to save whole topology state at a time.